Package.swift

```diff
@@ -14,23 +14,22 @@ let excludeFiles: String = []
 let package = Package(
     name: "Web3swift",
     platforms: [
-        .macOS(.v10_12), .iOS(.v11)
+        .macOS(.v12), .iOS(.v15)
```

Comment:
By doing this it will end support for iOS 14 right?

Reply:
Yes.

Reply:
that change comes from the Promise removal PR, nothing to do with the Swift Linting

Reply:
once 515 is merged in to unstable a bunch of the commits in these Swift Lint PRs will clear up

Reply:
> By doing this it will end support for iOS 14 right?
> 

We're definitely will not drop iOS 14 support. 

The lowest versions of the 3.0.0 framework release will be iOS 13 and macos 10.15.

Reply:
updated